ui/src/components/common/TopMenu.tsx

```diff
@@ -42,6 +44,14 @@ const TopMenu: React.FC<Props> = ({ title, notifications, onLogout, topMenuButto
                             </Button>
                         </Menu.Item>
                     )}
+                    {topMenuButtons &&
+                        <Menu.Item className='overflow-menu-item'>
+                            <OverflowMenu>
+                                {topMenuButtons?.map(b =>
+                                    <OverflowMenuEntry label={b.label} onClick={b.onClick}/>
```

Comment:
this'll need a 🗝️  

(`b.label` should suffice, if it's same to assume each label is unique)

Reply:
oops, thanks!